src/lib/core/backtrace.c

```diff
@@ -57,29 +57,34 @@ collect_current_stack(struct backtrace *bt, void *stack)
 	unw_context_t unw_ctx;
 	int rc = unw_getcontext(&unw_ctx);
 	if (rc != 0) {
-		say_error("unwinding error: unw_getcontext failed");
+		/*
+		 * XXX: turn this and other say_debug occurrences back to
```

Comment:
Please change the comment start to `FIXME(gh-7980)` (see also https://github.com/tarantool/tarantool/pull/7994#discussion_r1055363750), so it will be easier to find, and also please mention this in #7980.